TSQLLint.Lib/Parser/SqlRuleVisitor.cs

```diff
@@ -34,6 +34,13 @@ public void VisitRules(string sqlPath, Stream sqlFileStream)
             TextReader sqlTextReader = new StreamReader(sqlFileStream);
             var sqlFragment = fragmentBuilder.GetFragment(sqlTextReader, out var errors);
 
+            if (sqlFragment == null)
```

Comment:
reintroducing this guard. Get Fragment will be null if the sql itself is unparseable (not just syntactically invalid)